etc/windows/SyncThingSetup.nsi

```diff
@@ -71,7 +71,7 @@ Section "Command Line Interface" SecSyncThingCLI
 
   SetOutPath "$INSTDIR"
   
-  File /r "${SOURCEPATH}\syncthing-cli.exe"  
```

Comment:
Yeah it should be `syncthing-cli.exe`: it's in an optional section called "Command Line Interface"